src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses BioGRID data and maps the interaction types to BEL."""
+
+import pandas as pd
+
+import pybel.dsl
+from zipfile import ZipFile
+from bio2bel.utils import ensure_path
+from pybel import BELGraph
+
+from protmapper.uniprot_client import get_mnemonic
+
+# from ..constants import BIOGRID_ASSOCIATION_ACTIONS, BIOGRID_DECREASES_ACTIONS, BIOGRID_INCREASES_ACTIONS
+
+SEP = '\t'
+BIOGRID = 'biogrid'
+SOURCE = 'source'
+TARGET = 'target'
+RELATION = 'relation'
+PUBMED_ID = 'pubmed_id'
+MODULE_NAME = 'biogrid'
+VERSION = '3.5.183'
+BASE_URL = 'https://downloads.thebiogrid.org/Download/BioGRID/Release-Archive'
+URL = f'{BASE_URL}/BIOGRID-{VERSION}/BIOGRID-ALL-{VERSION}.mitab.zip'
+FILE = f'BIOGRID-ALL-{VERSION}.mitab.zip'
+
+
+#: Relationship types in BioGRID that map to BEL relation 'increases'
+BIOGRID_INCREASES_ACTIONS = {
+    'synthetic genetic interaction defined by inequality',
+    'additive genetic interaction defined by inequality',
+}
+
+#: Relationship types in BioGRID that map to BEL relation 'decreases'
+BIOGRID_DECREASES_ACTIONS = {
+    'suppressive genetic interaction defined by inequality',
+}
+
+#: Relationship types in BioGRID that map to BEL relation 'association'
+BIOGRID_ASSOCIATION_ACTIONS = {
+    'direct interaction',
+    'physical association',
+    'colocalization',
+    'association',
+}
+
+BIOGRID2BEL_FUNCTION_MAPPER = {
+    'direct interaction': '',
+    'suppressive genetic interaction defined by inequality': 'geneAbundance',
+    'physical association': '',
+    'colocalization': 'location',
+    'synthetic genetic interaction defined by inequality': 'geneAbundance',
+    'association': '',
+    'additive genetic interaction defined by inequality': 'geneAbundance'
+}
+
+
+def _load_file(module_name: str = MODULE_NAME, url: str = URL) -> str:
```

Comment:
does this need to be a function? it's just wrapping a call to another function, and it only gets used once, right?